R/get_source_extract_path.R

```diff
@@ -46,7 +46,7 @@ get_source_extract_path <- function(year,
     type == "AT" ~ "Alarms-Telecare-for-source",
     type == "CH" ~ "care_home_for_source",
     type == "CMH" ~ "CMH_for_source",
-    type == "Client" ~ "Client_for_source",
+    type == "Client" ~ "client_for_source",
```

Comment:
I guess the file name is now in lower case as I cannot fetch the file with the capital letter